pipe-cli/pipe.py

```diff
@@ -1623,13 +1623,15 @@ def cluster():
 @click.option('-p', '--interval', required=False, help='The time interval. This option shall have the following format:'
                                                        ' <N>m for minutes or <N>h for hours, where <N> is the required '
                                                        'number of minutes/hours. Default: 1m.')
+@click.option('-rt', '--report-type', required=False, help='Exported report type (case insensitive).'
```

Comment:
Please leave `csv` as default type to keep current behaviour as default. Set value using `default='CSV'`.

Reply:
Done